core/src/loader.rs

```diff
@@ -1089,13 +1090,17 @@ impl<'gc> Loader<'gc> {
                 .map(|u| u.to_string())
                 .unwrap_or(swf_url);
 
+            let cloned_spoofed_or_swf_url = spoofed_or_swf_url.clone();
             let mut movie =
                 SwfMovie::from_data(&body, spoofed_or_swf_url, None).inspect_err(|_error| {
                     player
                         .lock()
                         .unwrap()
                         .ui()
-                        .display_root_movie_download_failed_message(true);
+                        .display_root_movie_download_failed_message(
+                            true,
+                            cloned_spoofed_or_swf_url,
```

Comment:
cleaner to just put `spoofed_or_swf_url.clone()` here rather than creating a new variable, no?

Reply:
Not familiar enough with Rust to know if it's needed here, but the earlier clone needed to be outside the closure because the closure took ownership of the variable. Don't know if that occurs here too.

Reply:
I think you're right here though, always hard to keep track of this stuff in Rust when I'm more familiar with languages without this.

Reply:
Thanks, I could do this for the approach this PR took before, but I changed the approach instead. See the new code.